modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/servlet/DDMFormFunctionsServlet.java

```diff
@@ -64,58 +64,59 @@ protected void doGet(
 			HttpServletResponse httpServletResponse)
 		throws IOException, ServletException {
 
-		Map<String, DDMExpressionFunction> ddmExpressionFunctions = null;
+		Map<String, DDMExpressionFunctionFactory> ddmExpressionFunctions = null;
 
-		try {
-			ddmExpressionFunctions = getDDMExpressionFunctions();
+		ddmExpressionFunctions = getDDMExpressionFunctions();
 
-			String languageId = ParamUtil.getString(
-				httpServletRequest, "languageId");
+		String languageId = ParamUtil.getString(
+			httpServletRequest, "languageId");
 
-			JSONArray jsonArray = toJSONArray(
-				ddmExpressionFunctions.entrySet(),
-				LocaleUtil.fromLanguageId(languageId));
+		JSONArray jsonArray = toJSONArray(
+			ddmExpressionFunctions.entrySet(),
+			LocaleUtil.fromLanguageId(languageId));
 
-			httpServletResponse.setContentType(ContentTypes.APPLICATION_JSON);
-			httpServletResponse.setStatus(HttpServletResponse.SC_OK);
+		if (jsonArray == null) {
```

Comment:
Hey @carolmariaabb,

From what I could see, the `jsonArray` variable won't be `null` according to the implementation of the method `toJSONArray`, in light of this can you please tell me in which use case the `jsonArray` will be `null`? Am I missing something?